data/tribes/initialization/atlanteans/units.lua

```diff
@@ -197,6 +197,31 @@
 --
 --    **productionsite_workers_coming**: The string to display over this tribe's productionsites when more than 1 worker is coming.
 --
+--    **soldier_context**, **soldier_0_sg**, **soldier_0_pl**, **soldier_1_sg**, **soldier_1_pl**, **soldier_2_sg**, **soldier_2_pl**,
+--    **soldier_3_sg**, **soldier_3_pl**: The pgettext context for militarysites' soldier strings, and the corresponding **unlocalized**
+--    singular and plural strings. Additionally, it is necessary to specify the ``npgettext`` calls for all four types of soldier strings
+--    so as to ensure that all keys appear in the PO files (the ``npgettext`` results are unused). Example:
```

Comment:
please reword the brackets (the ``npgettext`` results (i.e. the translated pluralized string is not used but by a call in C++)
Please add: the format must be UNUSED_soldier_X = npgettext(soldier_context, soldier_X_sg, soldier_X_pl, 0),
where ix is going from 0 to 3 to matchg the c++ call.